src/views/ont/ONTRuns.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <DataFetcher :fetcher="fetchOntRuns">
+    <div class="clearfix">
+      <traction-pagination
+        v-model="currentPage"
+        class="float-right"
+        :total-rows="runs.length"
+        :per-page="perPage"
+        aria-controls="run-index"
+      ></traction-pagination>
+
+      <traction-form-group label-cols-lg="1" label="Per Page" label-for="input-per-page">
+        <traction-input id="input-per-page" v-model="perPage" trim class="w-25"></traction-input>
+      </traction-form-group>
+    </div>
+
```

Comment:
Is filtering needed for ont runs?

Reply:
It wasn't in the scope/ acceptance criteria, or mentioned when we did UAT with Alex. But probably in the future?

Reply:
Yeah probably fine without for time being but will definitely be needed in the future